### PR TITLE
Change indepency of fabric version

### LINKED ISSUE #91: Update fabric version
## Overview (Required)
Title says everything.
https://github.com/DroidKaigi/conference-app-2018/blob/master/build.gradle#L18

### PR DESCRIPTION
**Pull request by ycrack**
## Issue
- close #91

## Overview (Required)
- Change version number from 1.24.4 to 1.+

### CODE REVIEW

**Comment by konifar** [comment]
Thanks! 👀

File: build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ buildscript {
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
         classpath 'com.google.gms:google-services:3.1.0'
         classpath "gradle.plugin.org.jlleitschuh.gradle:ktlint-gradle:3.0.0"
-        classpath 'io.fabric.tools:gradle:1.24.4'
+        classpath 'io.fabric.tools:gradle:1.+'
```
Comment (konifar) [inline build.gradle]: I think it's better to specify the concrete version number to prevent including unexpected bug.
What do you think?

Comment (ycrack) [reply build.gradle]: I changed according to the official reference(https://docs.fabric.io/android/crashlytics/build-tools.html). Is it wrong?

Comment (konifar) [reply build.gradle]: Not wrong! but in my opinion, this official way is not good 😅

Comment (ycrack) [reply build.gradle]: The latest version seems to be 1.25.1.

**Comment by konifar** [comment]
LGTM! Thanks for quick fix!

**Comment by ycrack** [comment]
I found a mistake...
False: indepency
True: dependency
Plz forgive me! :)

**Pull request merged**